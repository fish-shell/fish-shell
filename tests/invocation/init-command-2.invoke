-C 'echo init-command' -C 'echo 2nd init-command'
