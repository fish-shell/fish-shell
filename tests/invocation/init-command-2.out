init-command
2nd init-command
